calibr8/utils.py

```diff
@@ -112,57 +112,18 @@ def istensor(input:object) -> bool:
     return False
 
 
-def plot_norm_band(ax, independent, mu, scale):
-    """Helper function for plotting the 68, 90 and 95 % likelihood-bands of a Normal distribution.
+def plot_continuous_band(ax, independent, model, residual_type: typing.Optional[str]=None):
+    """Helper function for plotting the 84, 95 and 97.5 % likelihood-bands of a univariate distribution.
```

Comment:
Could be independent of `model` objects? And if so, is that desireable? 🤔

Reply:
I found it hard to not use it here because so much needs so be done with both the scipy dist and the predict function and then the signature would be really complex... If you are ok with it, let's leave it like this.

Reply:
Yes we can leave it for now.

I think generally one could make it a more "functional" pattern that's more flexible, but maybe that's something we could contribute to ArviZ (similarly `plot_gp_dist` should probably be moved over to ArviZ and become more flexible now that I'm thinking about it..)